internal/databases/mongo/backup_list_handler.go

```diff
@@ -1,35 +1,177 @@
 package mongo
 
 import (
+	"encoding/json"
+	"fmt"
 	"io"
+	"sort"
+	"strings"
+	"text/tabwriter"
+	"time"
 
+	"github.com/jedib0t/go-pretty/table"
 	"github.com/wal-g/tracelog"
-	"github.com/wal-g/wal-g/internal/databases/mongo/archive"
+	"github.com/wal-g/wal-g/internal"
+	"github.com/wal-g/wal-g/internal/databases/mongo/binary"
+	"github.com/wal-g/wal-g/internal/databases/mongo/models"
+	"github.com/wal-g/wal-g/pkg/storages/storage"
 )
 
-// HandleBackupsList prints current backups.
-func HandleBackupsList(downloader archive.Downloader,
-	listing archive.BackupListing,
-	output io.Writer,
-	verbose bool) error {
-	backupTimes, _, err := downloader.ListBackups()
+type BackupDetail struct {
+	BackupName string    `json:"backup_name"`
+	ModifyTime time.Time `json:"modify_time"`
+
+	TsStart models.Timestamp `json:"ts_start"`
+	TsEnd   models.Timestamp `json:"ts_end"`
+
+	StartLocalTime  time.Time `json:"start_local_time"`
+	FinishLocalTime time.Time `json:"stop_local_time"`
+
+	UncompressedSize int64  `json:"uncompressed_size,omitempty"`
+	CompressedSize   int64  `json:"compressed_size,omitempty"`
+	Hostname         string `json:"hostname,omitempty"`
+
+	IsPermanent bool        `json:"is_permanent"`
+	UserData    interface{} `json:"user_data,omitempty"`
+}
+
+func NewBinaryBackupDetail(backupTime internal.BackupTime, sentinel binary.MongodBackupMeta) *BackupDetail {
+	return &BackupDetail{
+		BackupName:       backupTime.BackupName,
+		ModifyTime:       backupTime.Time,
+		TsStart:          sentinel.MongodMeta.TsStart,
+		TsEnd:            sentinel.MongodMeta.TsEnd,
+		StartLocalTime:   sentinel.StartLocalTime,
+		FinishLocalTime:  sentinel.FinishLocalTime,
+		UncompressedSize: sentinel.UncompressedDataSize,
+		CompressedSize:   sentinel.CompressedDataSize,
+		Hostname:         sentinel.Hostname,
+		IsPermanent:      sentinel.Permanent,
+		UserData:         sentinel.UserData,
+	}
+}
+
+func NewLogicalBackupDetail(backupTime internal.BackupTime, sentinel models.Backup) *BackupDetail {
+	return &BackupDetail{
+		BackupName:      backupTime.BackupName,
+		ModifyTime:      backupTime.Time,
+		TsStart:         sentinel.MongoMeta.Before.LastMajTS,
+		TsEnd:           sentinel.MongoMeta.After.LastMajTS,
+		StartLocalTime:  sentinel.StartLocalTime,
+		FinishLocalTime: sentinel.FinishLocalTime,
+		//UncompressedSize: sentinel.UncompressedDataSize,
```

Comment:
Uncomment these lines?

Reply:
I can't do it, because for logic backup I should implement it. I don't want to do it right now (in initial mvp).

Reply:
OK, I would suggest to add comment above to avoid confusion

Reply:
I fix it